lib/locales/uk.yml

```
@@ -55,6 +55,8 @@ uk:
       name_with_middle:
         - "#{first_name} #{last_name} #{last_name}"
 
+    country_code: ['38']
```

let's add the correct code instead:
```suggestion
    country_code: ['380']
```